### ISSUE TITLE
I get error while installing 

### ISSUE THREAD

**Issue by mayurrajurkar** [fossology/fossology#1934]
### Description

Please describe your issue in few words here.

#### How to reproduce

Describe the bug and list the steps you used when the issue occurred.

### Versions

* Last commit id on master:
* Operating System (lsb_release -a):

### Logs

Any logs (if any) generated in

#### FOSSology logs

Logs generated under /var/log/fossology/fossology.log

#### Apache logs

Logs generated under /var/log/apache2/error.log

#### Job logs

Logs generated under Geeky Scan Details (Job history => click on agent's job id)

**Comment by itssingh** [comment]
Are you following this video https://www.youtube.com/watch?v=q12KwmPYZG4? Also can you give more context about things you have done and on which os are you trying to set up?

**Comment by avinal** [comment]
Hello @mayurrajurkar Please attach the respective logs, either screenshot or log files.

**Comment by avinal** [comment]
Hello @mayurrajurkar Please attach the respective logs, either screenshot or log files.

**Comment by th** [comment]
Same error message here. This is on Windows with fossology from git, commit 87dd34ddc0d92e050cc45a43300f4dba1a011cf4.

 => CACHED [scheduler builder 18/21] RUN mkdir -p /fossology/dependencies-for-runtime  && cp -R /fossology/src /f  0.0s
 => ERROR [scheduler builder 19/21] RUN DEBIAN_FRONTEND=noninteractive apt-get update  && DEBIAN_FRONTEND=noninte  2.2s
------
 > [scheduler builder 19/21] RUN DEBIAN_FRONTEND=noninteractive apt-get update  && DEBIAN_FRONTEND=noninteractive /fossology/utils/fo-installdeps --build -y  && DEBIAN_FRONTEND=noninteractive /fossology/install/fo-install-pythondeps --build -y  && rm -rf /var/lib/apt/lists/*:
0.360 Get:1 http://deb.debian.org/debian bookworm InRelease [151 kB]
0.432 Get:2 http://deb.debian.org/debian bookworm-updates InRelease [55.4 kB]
0.460 Get:3 http://deb.debian.org/debian-security bookworm-security InRelease [48.0 kB]
0.487 Get:4 http://deb.debian.org/debian bookworm/main amd64 Packages [8788 kB]
0.907 Get:5 http://deb.debian.org/debian bookworm-updates/main amd64 Packages [13.8 kB]
0.908 Get:6 http://deb.debian.org/debian-security bookworm-security/main amd64 Packages [169 kB]
1.695 Fetched 9225 kB in 1s (6603 kB/s)
1.695 Reading package lists...
2.152 /usr/bin/env: 'bash\r': No such file or directory
2.152 /usr/bin/env: use -[v]S to pass options in shebang lines
------
failed to solve: process "/bin/sh -c DEBIAN_FRONTEND=noninteractive apt-get update  && DEBIAN_FRONTEND=noninteractive /fossology/utils/fo-installdeps --build -y  && DEBIAN_FRONTEND=noninteractive /fossology/install/fo-install-pythondeps --build -y  && rm -rf /var/lib/apt/lists/*" did not complete successfully: exit code: 127

**Comment by th** [comment]
Tag 4.4.0 doesn't build, either. Same error message. Probably Windows vs. Unix line endings?

**Comment by th** [comment]
Downloading the 4.4.0 zip file and doing docker compose build works. git on Windows is usually configured to check out as CRLF and commit as LF. That is usually the right thing to do (and works with Dockerfiles), but since fossology has several layers of bash scripts in post install, that git configuration is interfering here.

I will continue with my zip download for now, but people wanting the git version could change the setting.

`git config --global core.autocrlf false` is probably not a great idea, but .gitattributes can config per extension:
`*.sh text eol=cr`

**Comment by avinal** [comment]
@th yes it seems like a line ending issue. Adding a gitattributes file is a good idea.
